content/en/security/application_security/troubleshooting.md

```diff
@@ -564,6 +564,28 @@ See the Application Security [set up documentation][11] to validate you you are
 
 Ensure the `DD_INSTRUMENTATION_TELEMETRY_ENABLED` environment variable (`DD_TRACE_TELEMETRY_ENABLED` for NodeJS) is set to `true`, or the corresponding system property for your language is enabled. For example in Java: `-Ddd.instrumentation.telemetry.enabled=true`
 
+## Disabling Threat Detection and Protection
+
+To disable ASM, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration, and restart your service.
+
+If no `DD_APPSEC_ENABLED=true` environment variable is set for your service:
+* If it's a PHP service: explicitly set the environment variable to `DD_APPSEC_ENABLED=false`, and restart your service.
+* If ASM was activated in one-click: go to [ASM service configuration][15], select **Services with Threat Management Capabilities**, click on your service and then click **Deactivate**.
```

Comment:
```suggestion
* If ASM was activated in one click: go to [ASM service configuration][15], select **Services with Threat Management Capabilities**, click on your service, and then click **Deactivate**.
```

`one-click` is an adjective, `one click` is the noun form, e.g. _the one-click install method enabled me to install the software with one click_.

You'll also see this pattern repeated for `real time`/`real-time`, `command line`/`command-line`, etc.